operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -66,13 +65,22 @@ keys.
 Passwordless SSH with the provided keys will be setup once NCN personalization
 runs on the NCNs.
 
-### Option 3: Disable CSM-provided Passwordless SSH
+###  Disable CSM-provided Passwordless SSH
 
-Local site security requirements may preclude use of passwordless SSH access. If
-this is the case, remove or comment out the invocation of the
-`trust-csm-public-keys` role in Ansible plays in the configuration repositories
-of the environments where it is configured. By default, the HPE Cray Operating
-System (COS) and User Access Node (UAN) configurations enable passwordless SSH.
+Local site security requirements may preclude use of passwordless SSH access between
+management nodes. A variable has been added to the associated ansible roles that will
+allow you to disable passwordless ssh setup to any or all nodes.
+
+    ncn-w001:~ # grep csm_passwordless_ssh_enabled roles/trust-csm-ssh-keys/defaults/main.yaml
```

Comment:
Also, this overall command example should be enclosed by lines like these:
\```

Reply:
Done.